docs/src/neuralestimators_amorized.md

```diff
@@ -0,0 +1,261 @@
+# Neural Parameter Estimation
+
+Neural parameter estimation provides a likelihood-free approach to parameter recovery, especially useful for models with computationally intractable likelihoods. This method is based on training neural networks to learn the mapping from data to parameters. Once trained, these networks can perform inference rapidly across multiple datasets, making them particularly valuable for models like the Leaky Competing Accumulator (LCA).
+
+Below, we demonstrate how to estimate parameters of the LCA model using the [NeuralEstimators.jl](https://github.com/msainsburydale/NeuralEstimators) package.
+
+## Example
+
+We'll estimate parameters of the LCA model, which is particularly challenging due to its complex dynamics where parameters like leak rate (λ) and lateral inhibition (β) can be difficult to recover.
+
+## Load Packages
+
+```julia
+using NeuralEstimators
+using SequentialSamplingModels
+using Flux
+using Distributions
+using Random
+
+using Plots
+
+Random.seed!(123)
+```
+
+## Define Parameter Bounds
+
+```julia
+# Define parameter bounds for the LCA model
+const ν_min, ν_max = 0.1, 4.0      # Drift rates
+const α_min, α_max = 0.5, 3.5      # Threshold
+const β_min, β_max = 0.0, 0.5      # Lateral inhibition
+const λ_min, λ_max = 0.0, 0.5      # Leak rate
+const τ_min, τ_max = 0.1, 0.5      # Non-decision time
+```
+
+## Define Parameter Sampling
+
+Unlike traditional Bayesian approaches, simulation based inference methods require us to define a prior sampling function to generate training data. We will use this function to sample a range of parameters for training:
+
+```julia
+# Function to sample parameters from priors
+function sample(K::Integer)
+    ν1 = rand(Gamma(2, 1/1.2), K)  # Drift rate 1
+    ν2 = rand(Gamma(2, 1/1.2), K)  # Drift rate 2
+    α = rand(Gamma(10, 1/6), K)    # Threshold
+    β = rand(Beta(1, 5), K)        # Lateral inhibition
+    λ = rand(Beta(1, 5), K)        # Leak rate
```

Comment:
Its slightly ugly, but one way to generate `Vector{Float32}` for efficiency is to use `rand(Gamma(2f0, 1/1.2f0), K)`

Reply:
You could simplify with `rand(Gamma(2, 1/1.2f0), K)`. Since the first argument is an integer, it will promote the `Float32`